packages/teams-js/src/internal/handlers.ts

```diff
@@ -182,6 +183,27 @@ export function handleThemeChange(theme: string): void {
   }
 }
 
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function registerHostToAppPerformanceMetricsHandler(
```

Comment:
nit: Do you think we can give the app devs the option to unregister?

Reply:
Good question. I wouldn't think there is a need for unregistering for now. Normally apps want to recollect the data or none at all, which they wouldn't register the handler in the first place.